### DISCUSSION TITLE
`chrono` vs `time`

### DISCUSSION THREAD

**Discussion by tyt2y3** [SeaQL/sea-orm#959] [Ideas]
`chrono` now have new releases and new maintainers! https://github.com/chronotope/chrono/releases/tag/v0.4.20

They promised to solve (dump) the `time` 0.1 dependency.

What is the community's thought on this?

I am considering whether we should change to prefer `time` across our documentation and examples.

**Comment by billy1624** [comment]
Personally, from a maintainer point of view, I'd favour `time` over `chrono` as it's widely adopted, much more downloads on crates.io and better maintained.
- https://crates.io/crates/chrono
- https://crates.io/crates/time

**Reply by lidh15** [reply]
but also more difficult to use I think.

**Reply by billy1624** [reply]
Hey @lidh15, you got some experience to share?
